### ISSUE TITLE
Remove git integration

### ISSUE THREAD

**Issue by phobologic** [remind101/emp#15]
https://github.com/remind101/empire/issues/604#issuecomment-124172208

It looks like we still do something when you are in a git repo.  SHould probably remove that.

**Comment by rgabo** [comment]
It would be great if emp picked up `git remote` or even just `dirname` so that we can skip the `-a <app>` in every single invocation. Directory names == project names == docker images == Empire apps. Just let me tell `emp` to assume that if I'm in `acme-inc`, I want to work with `acme-inc`.

**Comment by ejholmes** [comment]
@rgabo yeah, I agree. I think `dirname` would be the most straightforward thing to do. Git remote is a little confusing since Empire doesn't support git pushes.
